Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -103,3 +103,46 @@
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
 fastSim.toReplaceWith(validation_prod,_validation_prod_fastsim)
 
+from SimGeneral.MixingModule.mixObjects_cfi import theMixObjects
+from SimGeneral.MixingModule.mixPoolSource_cfi import *
+from SimGeneral.MixingModule.digitizers_cfi import *
+
+mixNoPileUp = cms.EDProducer("MixingModule",
```

Comment:
Why is additional instance of `MixingModule` needed? IIRC `MixingModule` is already run in playback mode in jobs with VALIDATION step. I also don't see `mixNoPileUp` product label used in this PR nor in `master`.

Reply:
@makortel : Thanks for the correction. Indeed we don't need an additional instance of the mixing module. The test workflows succeed with it removed. I have modified the PR.

Reply:
@jfernan2 I believe the tests need to be restarted? Thanks!